### PR TITLE
Update install instructions ahead of UGM

### PR DESCRIPTION
**Pull request by IAlibay**
----
:books: Documentation preview :books:: https://mdanalysisuserguide--324.org.readthedocs.build/en/324/

### CODE REVIEW

File: doc/source/contributing_code.rst [hunk 1]
```diff
@@ -239,12 +239,12 @@ First we need to install dependencies. You'll need a mix of conda and pip instal
...
-          'tidynamics>=1.0.0'
+          'tidynamics>=1.0.0' \
+          'mda-xdrlib'
 
         # documentation dependencies
-        conda install -c conda-forge sphinx pybtex pybtex-docutils \
-            sphinxcontrib-bibtex mdanalysis-sphinx-theme sphinx-sitemap
-        python -m pip install
+        conda install -c conda-forge 'mdanalysis-sphinx-theme>=1.3.0'
+        python -m pip install docutils sphinx-sitemap sphinxcontrib-bibtex pybtex pybtex-docutils
```
Comment (IAlibay) [inline doc/source/contributing_code.rst:247]: This is more closely replicating what CI does - I'm not sure if they are all conda installable, in which case we should just do that for everything?

**Comment by IAlibay** [comment]
We should make sure that we have everything shored up for the hackathon, might be worth cycling CI everywhere the night before.

File: doc/source/contributing_code.rst [hunk 2]
```diff
@@ -239,12 +239,12 @@ First we need to install dependencies. You'll need a mix of conda and pip instal
...
           'scikit-learn' \
-          'tidynamics>=1.0.0'
+          'tidynamics>=1.0.0' \
+          'mda-xdrlib'
 
         # documentation dependencies
-        conda install -c conda-forge sphinx pybtex pybtex-docutils \
-            sphinxcontrib-bibtex mdanalysis-sphinx-theme sphinx-sitemap
-        python -m pip install
+        conda install -c conda-forge 'mdanalysis-sphinx-theme>=1.3.0'
```
Comment (lilyminium) [inline doc/source/contributing_code.rst]:
```suggestion
        conda install -c conda-forge 'mdanalysis-sphinx-theme>=1.3.0' docutils sphinx-sitemap sphinxcontrib-bibtex pybtex pybtex-docutils
```

File: doc/source/contributing_code.rst (hunk 1 above)
Comment (lilyminium) [reply doc/source/contributing_code.rst:247]: Yes, I think we can do everything with conda. Or at least I can do anything with micromamba

**Comment by lilyminium** [comment]
Thanks for opening this @IAlibay!!

**Comment by lilyminium** [comment]
I actually think installing sphinx pulls in a bunch of dependencies like bibtex, but probably not going to iterate through minimalist builds now

**Comment by IAlibay** [comment]
> I actually think installing sphinx pulls in a bunch of dependencies like bibtex, but probably not going to iterate through minimalist builds now

Yeah, this was very much a "so folks can get dev environments going" - especially since we're recommending doctest issues

**Review by lilyminium — APPROVED** [review]
LGTM!

**Pull request merged**